### PR TITLE
Update doc for 3.0 and 3.1

### PR DESCRIPTION
**Pull request by anupriya0703**
The following changes have been made in this PR-
-  3.0.x section has been added under the drop-down
-  'Main' with 3.1.0 contents has been uploaded.

Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md [hunk 1]
```diff
@@ -0,0 +1,2506 @@
...
+
+**Size:** cStor supports thin provisioning by default, which means you can request any size of the volume through the PVC and get it provisioned. Resize of the volume is not fully supported through the OpenEBS control plane in the current release and it is under active development, see [roadmap](/docs/concepts/cstor#cstor-roadmap) for more details. Hence it is recommended to give good amount of buffer to the required size of the volume so that you don't need to resize immediately or in a very short time period. 
+
+The following shows the example PVC configuration for a Deployment and a StatefulSet application which uses a configured StorageClass to provision a cStor Volume. The provided StorageClass should have been configured with StoragePoolClaim property, so the cStor Volume will be provisioned on the StoragePools associated to the StoragePoolClaim.
+
+#### **Example configuration for requesting OpenEBS volume for a Deployment**
```
Comment (Pallavi-PH) [inline docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md]:
```suggestion
#### Example configuration for requesting OpenEBS volume for a Deployment
```

Heading texts are auto converted to bold

File: docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md [hunk 2]
```diff
@@ -0,0 +1,2506 @@
...
+ kubectl describe blockdevice <blockdevicename> -n <openebs_namespace>
+```
+
+Example:
+
+```
+kubectl describe blockdevice blockdevice-1c10eb1bb14c94f02a00373f2fa09b93 -n openebs 
+```
+
+From the output, you will get the hostname and other blockdevice details such as State,Path,Claim State,Capacity etc.
```
Comment (Pallavi-PH) [inline docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md]:
```suggestion
From the output, you will get the hostname and other blockdevice details such as State, Path, Claim State, Capacity, etc.
```

File: docs/versioned_docs/version-3.0.x/faqs/cStor.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+  filterconfigs:
+    - key: path-filter
+      name: path filter
+      state: true
+      include: ""
+      exclude: "/dev/loop,/dev/fd0,/dev/sr0,/dev/ram,/dev/dm-,/dev/md,/dev/rbd,/dev/zd"
+```
+
+[Go to top](#top)
+### What is the difference between cStor Pool creation using manual method and auto method?
```
Comment (Pallavi-PH) [inline docs/versioned_docs/version-3.0.x/faqs/cStor.md:214]:
```suggestion
[Go to top](#top)

### What is the difference between cStor Pool creation using manual method and auto method?
```

File: docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md (hunk 1 above)
Comment (anupriya0703) [reply docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md]: Have made the changes- https://github.com/openebs/website/pull/293/commits/6e79209b1999a3142f783cea507f33d65d6ed31f

File: docs/versioned_docs/version-3.0.x/faqs/cStor.md (hunk 3 above)
Comment (anupriya0703) [reply docs/versioned_docs/version-3.0.x/faqs/cStor.md]: The changes are made under- https://github.com/openebs/website/pull/293/commits/6e79209b1999a3142f783cea507f33d65d6ed31f

File: docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md (hunk 2 above)
Comment (anupriya0703) [reply docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md]: The changes are made under- https://github.com/openebs/website/pull/293/commits/6e79209b1999a3142f783cea507f33d65d6ed31f

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by Pallavi-PH — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by Pallavi-PH — APPROVED** [review]

**Pull request merged**